src/Razor/src/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X/ViewComponentTagHelperDescriptorFactory.cs

```diff
@@ -110,11 +110,11 @@ private bool TryFindInvokeMethod(INamedTypeSymbol type, out IMethodSymbol method
             if (string.Equals(selectedMethod.Name, ViewComponentTypes.AsyncMethodName, StringComparison.Ordinal))
             {
                 // Will invoke asynchronously. Method must not return Task or Task<T>.
-                if (Equals(returnType, _taskSymbol))
+                if (SymbolEqualityComparer.Default.Equals(returnType, _taskSymbol))
```

Comment:
These changes were forced upon me by an analyzer.

Reply:
Would `using static System.SymbolEqualityComparer.Default SymbolComparer;` work in these files?

Reply:
Nope. Doesn't let me reference static properties like that.

Reply:
Ah well, I had a typo anyhow. Could still use an alias but it wouldn't shorten things much e.g.
```
using SymbolComparer = SymbolEqualityComparer;
```